src/org/zaproxy/zap/authentication/AuthenticationMethodType.java

```diff
@@ -149,22 +151,23 @@ public abstract AuthenticationMethod loadMethodFromSession(Session session, int
 	 * @param authMethod the auth method to persist
 	 * @throws UnsupportedAuthenticationMethodException the unsupported authentication method
 	 *             exception
+	 * @throws DatabaseException if an error occurred while persisting the authentication method
 	 */
 	public abstract void persistMethodToSession(Session session, int contextId,
-			AuthenticationMethod authMethod) throws UnsupportedAuthenticationMethodException, DatabaseException;
+			AuthenticationMethod authMethod) throws DatabaseException;
 
 	/**
 	 * Export the specified method to the configuration
-	 * @param config
-	 * @param authMethod
+	 * @param config the configurations where to export/save the authentication method
+	 * @param authMethod the authentication method to be exported
 	 */
 	public abstract void exportData(Configuration config, AuthenticationMethod authMethod);
 
 	/**
-	 * Import the method from the confuguration
-	 * @param config
-	 * @param authMethod
-	 * @throws ConfigurationException
+	 * Import the method from the configuration
+	 * @param config the configurations from where to import/load the authentication method
+	 * @param authMethod where to set the imported authentication method data
```

Comment:
No, the `authMethod` is where the imported data (read from the Configuration) is set.